social_core/backends/okta.py

```diff
@@ -0,0 +1,60 @@
+"""
+Okta OAuth2 and OpenIdConnect:
+    https://python-social-auth.readthedocs.io/en/latest/backends/okta.html
```

Comment:
Didn't you forget to add this page?

Reply:
No. There's a pull request to the documentation open: https://github.com/python-social-auth/social-docs/pull/56.